content/ops/security-ir.md

```diff
@@ -0,0 +1,228 @@
+
+The first responder at this point becomes *Incident Commander* (IC), and carries out the next steps in the response. The IC's responsibility is coordination, not necessarily investigation. The IC's primary role is to guide the process. The first responder may remain IC throughout the process, or they may hand off IC duties later in the process.
+
+The first step for the IC is to create an issue in the [`security-incidents`](https://github.com/18f/security-incidents) github repo. Copy the following template to create the issue:
+
+```
+Short description of what's going on
+
+- **Status**: investigating
+- **Severity**: unknown
+- **Reporter**: who initially reported the issue
+- **IC**: Your Name
+- **Responders**: Any other responders
+
+Any extra details about the issue can go here.
+```
+
+The IC is responsible for keeping this issue up-to-date as investigation and remediation progresses. Everyone involved in the issue (*responders*) should leave notes as comments on the issue.
+
+#### Comms at the Initiate phase
+
+Note that at this point the issue's status is "investigating" - we haven't confirmed that it's really an issue yet. So, we should actually refer to this as just an "event" at this point; it doesn't become an "incident" until we've confirmed it. 
+
+At this phase, communications should follow these guidelines: 
+
+- The IC should inform GSA IT and our devops teams of the investigation by emailing `gsa-ir@gsa.gov` and `devops@gsa.gov`. **This needs to happen within 1 hour.**
+- Updates should be posted to the Github issue.
+- Real-time chat should happen in `#incident-response`
+- The IC may start a Google Hangout and/or create Google Docs so that responders can share sensitive information not suitable for sharing in Github or Slack.
+
+### Assess
+
+The next step is to assess the issue. We need to answer two questions:
+
+- Is this an incident (i.e., did the thing we suspect happen actually happen?)
+- If so, how severe is it? (This will determine how our response proceeds.)
+
+To answer these questions, the IC should form a response team by DM'ing people in Slack. The response team should work to confirm the issue and assess its impact. 
+
+If the issue turns out to be a false alarm, the IC should update the ticket, setting the status to "false alarm", and closing the issue. 
+
+If the issue is valid, the team should assess its impact and determine an initial severity following the incident severity guide below. (Note that the severity can change over the lifespan of the incident, so it's OK to determine the initial severity fairly quickly.)
+
+Once this is done, the IC should update the ticket, noting:
+
+- Status: "confirmed"
+- Severity: High/Med/Low
+- Any new/changed responders
+
+At this point, the IC should post an initial situation report ("sitrep") confirming the incident, summarizing what's going on, identifying the IC, and linking to the issue. Here's an example sitrep:
```

Comment:
Since the sitrep is referring to the issue, where are we expecting it to get "posted" outside of the issue? Or is the issue reference in the sitrep intended to act as an idempotent key for linking messages related to the event/incident from different places?